conductr_cli/data_structure_utils.py

```diff
@@ -0,0 +1,11 @@
+from collections import OrderedDict
+
+
+def sort_dict(d):
```

Comment:
Needed to sort a dictionary recursively alphabetically by key. An `OrderedDict` only sorts the keys of the root level.